bundles/io/pom.xml

```diff
@@ -18,6 +18,7 @@
   <modules>
     <module>org.openhab.io.transport.serial</module>
     <module>org.openhab.io.jetty</module>
+	<module>org.openhab.io.certificate.generator</module>
```

Comment:
It seems, that currently spaces are used to indent.
Could you change your tab to spaces?